client/src/service/getChat.js

```diff
@@ -0,0 +1,9 @@
+import makeRequest from '../utils/makeRequest';
+
+const getLink = async (pin) => {
+  return makeRequest(`/chat-link/${pin}`, {
+    method: 'GET'
+  });
+};
+
+export default getLink;
```

Comment:
As it's a default export, please keep the file name the same.
https://sonarcloud.io/project/issues?id=muke1908_chat-e2ee&pullRequest=167&resolved=false&types=CODE_SMELL